### PR TITLE
feat: styled toastify toasts

### PR DESCRIPTION
**Pull request by MicroCBer**

### CODE REVIEW

**Comment by NWYLZW** [comment]
Show a picture for  the new style?

**Comment by MicroCBer** [comment]
image
image

File: src/main.scss [hunk 1]
```diff
@@ -32,3 +32,59 @@ body {
...
+    --toastify-color-progress-success: var(--toastify-color-success);
+    --toastify-color-progress-warning: var(--toastify-color-warning);
+    --toastify-color-progress-error: var(--toastify-color-error);
+}
+
+.Toastify__progress-bar {
+    height: 3px;
+}
+
+button.Toastify__close-button {
```
Comment (NWYLZW) [inline src/main.scss:86]: https://github.com/Power-Playground/app/pull/18/files#diff-70a6c9cac6ea1a2d9a18701a285f545a1035c3b35437cdf87effc6741a001215R43

The above type seems to be similar to the following, can it be combined?

**Review by NWYLZW — APPROVED** [review]
LGTM

**Pull request merged**